### DISCUSSION TITLE
cannot set up navigation

### DISCUSSION THREAD

**Discussion by carolhuang1990** [squidfunk/mkdocs-material#2993] [Q&A]
Hi,
I am trying to set up navigation following by this article, but it doesn't work.
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/

For example, I set the followings in the file "mydoc.yml"
`theme:
  name: material
  features:
    - navigation.tabs`
After I execute the command `mkdocs serve`.
I cannot see any navigation tab displayed as the documentation described.

**Comment by squidfunk** [comment]
The config file needs to be named `mkdocs.yml`.

**Reply by carolhuang1990** [reply]
Sorry for my mistake in the last post.
Yes, my config file is named `mkdocs.yml`.
And I find it may be a problem with the compatibility with Chrome browser, not a problem of my website, but for all the websites that use "material" theme.

I am using Chrome Version 93.0.4577.63 (Official Build) (x86_64).

If the Chrome Page zoom is set to the default 100%：
- The home page is displayed, as the following figure shows:
https://drive.google.com/file/d/11e87upxj2nEUVVJmd8SDsF03rEWAVQk-/view?usp=sharing
- After click on the navigation icon, no navigation is displayed but a blank pop-up window, as the following figure shows:
https://drive.google.com/file/d/1PEETs8O59kriOmf6MmB-grdcH-KOzpTl/view?usp=sharing

If  I  set the Chrome Page zoom to 90%, I can see a correct home page and also the configured navigation tabs.
Also, no problem happens if I use Safari browser, I don't have to adjust the page zoom.

I am using MacOS Big Sur version 11.5.2 (20G95).

**Reply by squidfunk** [reply]
I just tested it on the official documentation with Chrome and Safari on macOS Catalina, and it works just fine, regardless of page zoom.

**Reply by carolhuang1990** [reply]
Thanks for your time.
I also tested on my colleagues' Macbook and Win PC.
The material theme works fine on their Chrome browsers.
I guess it is a problem of my Chrome.
Thanks! :)
